database/migrations/2018_07_24_111137_create_centre_users_table.php

```diff
@@ -24,6 +24,7 @@ public function up(): void
             $table->rememberToken();
             $table->integer('centre_id')->unsigned()->nullable();// Not all Users may have a CC e.g. Admins
             $table->timestamps();
+            $table->softDeletes();
```

Comment:
Should you be updating an existing migration?

Shouldn't this be a new migration.

Reply:
See above;

- this doesn't affect live, live has already run the migrations.
- this affect staging and local, which blow up because the model says soft deletes, but a later migration manipulates data before the softdeletes are added.
- can't add a new migration, as it will be run after the problem occurs.

Solution is to fold the later soft delete migration into the original creation, respecting intent but before the data manipulation happens.